E2E/tests/SPCounting.test.ts

```diff
@@ -0,0 +1,96 @@
+Feature('SP counting')
+
+const { I, login, game } = inject();
+import username = require("../test_data/usernames");
+import { ChessPiece } from "../test_data/chessPieces";
+import ChessTile = require("../test_data/chessTiles");
+import assertions = require("../assertions/assertions");
+
+Scenario('Individual final SP is rounded to closest value', async () => {
+  login.firstVoterLogin(username.user1);
+    let url = await I.grabCurrentUrl();
+    session(username.user2, () => {
+      login.voterLoginIntoCreatedGameRoom(url, username.user2);
+      game.vote(ChessPiece.queen, ChessTile.f5);
+    });
+    game.vote(ChessPiece.bishop, ChessTile.d3);
+    await assertions.checkIfTheSPIsRoundedCorrectlyForTheFirstPlayer();
+    await assertions.checkIfTheSPIsRoundedCorrectlyForTheSecondPlayer();
+});
+
+Scenario('Final SP is rounded to closest value when closest value is boundary value', async () => {
+  login.firstVoterLogin(username.user1);
+    let url = await I.grabCurrentUrl();
+    session(username.user2, () => {
+      login.voterLoginIntoCreatedGameRoom(url, username.user2);
+      game.vote(ChessPiece.queen, ChessTile.f5);
+    });
+    game.vote(ChessPiece.pawn, ChessTile.b3);
+    await assertions.checkIfTheFinalSPIsRoundedCorrectly();
```

Comment:
It is not clear what the test is expecting for, this should be written in some other way.

Reply:
I would expect to see something like this
```
assertions.finalSPAfterRoundingEqualsTo(5 (or whatever number is expected))
```

This makes the assertion reusable and the test more readable.